### PR TITLE
New package clickhouse-keeper

### PR DESCRIPTION
**Pull request by Felixoid**
Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
A dedicated small package for `clickhouse-keeper`.

### CODE REVIEW

**Comment by Felixoid** [comment]
Shouldn't be merged before #33664

**Comment by alesapin** [comment]
Going to add docker image in the next pull request.

**Comment by alesapin** [comment]
@mergifyio update

**Comment by alesapin** [comment]
@mergifyio update

File: packages/clickhouse-keeper.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+vendor: "ClickHouse Inc."
+homepage: "https://clickhouse.com"
+license: "Apache"
+section: "database"
+priority: "optional"
+
+conflicts:
+- clickhouse-server
+depends:
+- adduser
```
Comment (Felixoid) [inline packages/clickhouse-keeper.yaml:16]:
```suggestion
- adduser
suggests:
- clickhouse-keeper-dbg
```

File: packages/clickhouse-keeper-dbg.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+version: "${CLICKHOUSE_VERSION_STRING}"
+vendor: "ClickHouse Inc."
+homepage: "https://clickhouse.com"
+license: "Apache"
+section: "database"
+priority: "optional"
+
+conflicts:
+- clickhouse-server
+
```
Comment (Felixoid) [inline packages/clickhouse-keeper-dbg.yaml:15]: It doesn't conflict, actually

```suggestion

```

**Review by Felixoid — COMMENTED** [review]
We are good to go after some packaging tuning

File: src/Compression/CompressionFactory.cpp [hunk 3]
```diff
@@ -165,25 +165,36 @@ void registerCodecNone(CompressionCodecFactory & factory);
 void registerCodecLZ4(CompressionCodecFactory & factory);
 void registerCodecLZ4HC(CompressionCodecFactory & factory);
 void registerCodecZSTD(CompressionCodecFactory & factory);
+void registerCodecMultiple(CompressionCodecFactory & factory);
+
+
+/// Keeper use only general-purpose codes, so we don't need these special codecs
```
Comment (alesapin) [inline src/Compression/CompressionFactory.cpp:171]:
```suggestion
/// Keeper use only general-purpose codecs, so we don't need these special codecs
```

**Comment by alesapin** [comment]
Finally.

**Comment by alesapin** [comment]
Without symbols package size is about 6.8MB.

**Pull request merged**